src/Lykke.Service.ConfirmationCodes.Services/ConfirmationCodesService.cs

```diff
@@ -105,6 +97,22 @@ public async Task<bool> CheckAsync(string partnerId, string mobilePhone, string
         {
             return await _smsVerificationCodeRepository.CheckAsync(partnerId, mobilePhone, code);
         }
+        
+        public async Task<SmsCheckResult> CheckSmsAsync(string clientId, string mobilePhone, string code, string operation)
+        {
+            var result = new SmsCheckResult();
```

Comment:
For the sake of readability, I would rename it to smsCheck. `result.Result` looks bad.